src/shared/components/imageCard/imageCard.css

```diff
@@ -41,8 +34,8 @@
 /* Vertical space between cards for vertical layout */
 @media screen and (max-width: 667px) {
   .imageCard {
-    height: 400px;
-    width: 300px;
+    height: 400px !important;
+    width: 300px !important;
```

Comment:
Could you leave a comment as to why !important is necessary here?

Reply:
Sure. As mentioned in the pull request, one issue that arose was that passing the css as props overrode the media query that stacks the cards on smaller screens, so it was broken on mobile. My temporary fix was the add `!important` on the media query styles. Let me know if you think there is a better way around this.